Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -18,7 +18,8 @@
             name="google-site-verification"
             content="doAsS2f0kVQcQMjZcJuo2Pdj60GkhXI2iKbwfaUBImk"
         />
-
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js" integrity="sha512-3n19xznO0ubPpSwYCRRBgHh63DrV+bdZfHK52b1esvId4GsfwStQNPJFjeQos2h3JwCmZl0/LgLxSKMAI55hgw==" crossorigin="anonymous">
```

Comment:
instead of getting the script out of the internet on demand we can get it as a package and reference the node folder

Reply:
sure, I can try to bring it down to our node folder.

Reply:
@FuriousLlama I have generated a very small modernizr.js by adding just the es6 number feature and placed it under public folder, do you think it is the right place for it to be in?